Diagnostic/HandlerManifest.json

```diff
@@ -1,15 +1,16 @@
 [{
-  "name": "LinuxAzureDiagnostic",
-  "version": 1.0,
-  "handlerManifest": {
-   "installCommand": "diagnostic.py -install",
-    "uninstallCommand": "diagnostic.py -uninstall",
-    "updateCommand": "diagnostic.py -update",
-    "enableCommand": "diagnostic.py -enable",
-    "disableCommand": "diagnostic.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false,
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "LinuxDiagnostic",
+    "version": "2.3.9001",
```

Comment:
This `"version"` is the version for the schema of this file; not extension handler. And it should be a `float` with value `1.0` per spec.

Please read https://github.com/Azure/azure-marketplace/wiki/User%20Guide%20for%20writing%20extension%20handlers.pdf or see https://github.com/Azure/azure-docker-extension/blob/master/metadata/HandlerManifest.json as a reference.